interfaces/multimedia_board.py

```diff
@@ -27,9 +27,9 @@ def send_lighting_state(self, state):
                 "B",
                 (state,),
                 core.manifest["Multimedia"]["Ip"],
-                core.manifest["Multimedia"]["Port"],
+                core.UDP_OUTGOING_PORT,
             ),
-            True,
```

Comment:
While I don't know how necessary it is for Autonomy to send TCP packets to multimedia since there is practically no way a packet is going to be lost over a direct ethernet connection, it does seem like you may "_want_" to leave this as TCP and there doesn't seem to be a clear reason for the reversion. (And if you do want to use UDP, False is the default value of reliable and you don't _need_ to specify it. The fact you did makes me feel like maybe this was temporary or still being thought about)

Reply:
Same as above, some TCP issues with Multimedia board due to manifest. Will probably be fixed closer to final software REV, and is it is untested I won't change this for now.